### DISCUSSION TITLE
Guidance on hooking up zustand with HMR in react native

### DISCUSSION THREAD

**Discussion by mfbx9da4** [pmndrs/zustand#729] [Q&A]
Looking for guidance on hooking up HMR with zustand in react native.

https://reactnative.dev/blog/2016/03/24/introducing-hot-reloading

**Comment by chrisdrackett** [comment]
I've been trying to figure out how to get these to play nice as well. No luck so far, but I'll update if I make progress

**Reply by Nikola-Milovic** [reply]
Any update? :rofl:

**Comment by CamParry** [comment]
I'm also very interesting in this, I have a project that requires the store to be created in the same file as the component because I need a unique store per component, but this means any time I modify the component I lose all my state.
